api-specificatie/openapi.yaml

```diff
@@ -1054,519 +1003,215 @@ components:
+            zonder dat er op het moment van opname sprake was van een regulier brondocument
+            voor deze opname.
           type: boolean
-        documentdatum:
-          description: De datum van het document waarin de grondslag van het onderzoek wordt vastgelegd. Dit wordt vastgelegd in het attribuut documentdatum.
-          title: documentdatum
-          type: string
-          format: date
-          example: '2019-08-14'
-        documentnummer:
-          description: Het nummer van het document waarin de grondslag van het onderzoek wordt vastgelegd. Dit wordt vastgelegd in het attribuut documentnummer.
-          title: Documentnummer
+        geometrie:
+          $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/master/api-specificatie/lvbag/imbag/v20180601/geometrie.yaml#/components/schemas/PuntOfVlak'
+        hoofdAdresIdentificatie:
+          description: identificatie van het hoofdadres
           type: string
-          minLength: 1
-          maxLength: 40
-          example: 'AV-2019-08-19-00'
-
-    InOnderzoekStandplaats:
-      title: InOnderzoekStandplaats
-      type: string
-      enum:
-        - 'geometrie'
-        - 'status'
-        - 'heeft als hoofdadres'
-        - 'heeft als nevenadres'
-
-    Standplaats_links:
-      type: object
-      required:
-        - self
-      properties:
-        self:
-          $ref: '../../../../extern/haal-centraal/common-schemas.yaml#/HalLink'
-        adres:
-          $ref: '../../../../extern/haal-centraal/common-schemas.yaml#/HalLink'
-          description: 'Het aan de standplaats gerelateerde adres (link).'
-
-    Standplaats_embedded:
-      type: object
-      properties:
-        adres:
-          $ref: '#/components/schemas/Adres'
-          description: 'Het aan de ligplaats gerelateerde adres (object).'
-
-    StandplaatsInOnderzoek:
-      type: object
-      title: StandplaatsInOnderzoek
-      properties:
-        standplaats:
-          $ref: '#/components/schemas/Standplaats'
-        inonderzoek:
+          example: "01232067890123456"
+        nevenAdresIdentificaties:
+          description: identificatie(s) van het nevenadres of de nevenadressen
+          items:
+            type: string
+            example: "0123201234567890"
           type: array
+        pandIdentificaties:
+          description: identificatie(s) van het pand of de panden waar het verblijfsobject
+            deel van uitmaakt
           items:
-            $ref: '#/components/schemas/KenmerkStandplaatsInOnderzoek'
-          minItems: 1
-
-    StandplaatsHal:
+            type: string
+            example: "01234567890123456"
+          type: array
+        oppervlakte:
+          type: integer
+        status:
+          $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/master/api-specificatie/lvbag/imbag/v20180601/adresseerbaarObject.yaml#/AdresseerbaarObjectStatus'
```

Comment:
**adresseerbaarObject**.yaml in https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/master/api-specificatie/lvbag/imbag/v20180601/adresseerbaarObject.yaml

moet zijn **adresseerbaarobject**.yaml. GitHub is case-sensitive

Echter **AdresseerbaarObjectStatus** is niet gedefinieerd in het yaml bestand

Reply:
Nu aangepast naar een lokale enumeratie zoals die door Frank was gedefinieerd.